backend/api/groups.js

```diff
@@ -22,7 +24,7 @@ router.post("/", (req, res) => {
     .save()
     .then(() =>
       res.json({
-        group: newGroup,
+        group_id: group_id,
         message: "Added group successfully",
```

Comment:
It might be a bit confusing down the line to mix response data with a status message (on top of the status code) at the same object level. Can we just send back the created group, for consistency?